python/src/IterativeExtrema_doc.i.in

```diff
@@ -0,0 +1,74 @@
+%feature("docstring") OT::IterativeExtrema
+"Iterative minimum and maximum.
+
+Parameters
+----------
+dimension : int
+    Dimension of the input data
+
+Notes
+-----
+This class iteratively computes the min and the max of several :class:`~openturns.Point` or :class:`~openturns.Sample`.
+
+This class does not store the input data in memory.
```

Comment:
I am wondering whether it is a good idea to emphasize the OT types `Point` and `Sample` when in fact, pretty much any data container will do. Here is an alternative:

```suggestion
This class iteratively computes the minimum and maximum over an iteratively increasing
dataset without storing any data in memory.
```